google/oauth2/id_token.py

```diff
@@ -174,8 +174,8 @@ def verify_firebase_token(id_token, request, audience=None):
     )
 
 
-def fetch_id_token(request, audience):
-    """Fetch the ID Token from the current environment.
+def fetch_id_token_credentials(request, audience):
```

Comment:
Should we make the request parameter optional to keep consistent with `google.auth.default()`?